AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -653,6 +698,15 @@ - (void)displayImmediately
   [[self asyncLayer] displayImmediately];
 }
 
+- (void)recursivelyDisplayImmediately
```

Comment:
Hmm, that's exceedingly odd.  I'm OK with landing this approach, but will consider it a followup for me to debug that issue and consolidate the display mechanisms.  Thank you for trying it.